optuna/storages/rdb/storage.py

```diff
@@ -98,6 +98,15 @@ def create_new_study(self, study_name=None):
 
         return study.study_id
 
+    def delete_study(self, study_id):
+        # type: (int) -> None
```

Comment:
Please add a blank line after the type hint.

Reply:
Thank you! I fixed.